src/0.9.0/immudb/README.md

```diff
@@ -269,6 +274,24 @@ To deactivate an existing user, run `immuadmin user deactivate ro`
 
 To reactivate a deactivated user account, you can simply set user permission again. Run `immuadmin user set-permission ro readwrite`
 
+## State signature
+It's possible to sign each state with a private key.
+In this way clients and auditor can verify the immudb server identity.
+To launch immudb with signatures use `--signingKey` flag  `IMMUDB_SIGNING_KEY` environment variable.
```

Comment:
My proposal would be:

"To launch immudb with signatures, either use --signingKey` or flag `IMMUDB_SIGNING_KEY`"